MiguelAValdesG/Hackerrank/JavaList.java

```diff
@@ -0,0 +1,44 @@
+import java.util.LinkedList;
+import java.util.Scanner;
+
+
+public class JavaList {
+
+  public static void main(String[] args) {
+    Scanner readLineTerminal = new Scanner(System.in);
+    int numberOfElementsInList = readLineTerminal.nextInt();
+    final String ERROR = "ERROR!";
+    final String SPACE = " ";
+    StringBuilder response = new StringBuilder();
+    LinkedList<Integer> list = new LinkedList<>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices